src/components/Board.js

```diff
@@ -6,15 +6,18 @@ import PropTypes from 'prop-types';
 
 const generateSquareComponents = (squares, onClickCallback) => {
   // Complete this for Wave 1
+  let squaresFlat = squares.flat();
+  return squaresFlat.map((square) => {
+    return <Square key={square.id} id={square.id} value={square.value} onClickCallback={onClickCallback} />
+  })
```

Comment:
You can combine these into a single `flatMap`:

```suggestion
  return squares.flatMap((square) => {
    return <Square key={square.id} id={square.id} value={square.value} onClickCallback={onClickCallback} />
  })
```